backend/fast_stock_retrieval_1000.py

```
@@ -0,0 +1,369 @@
+    ordered = []
+    for s in symbols:
+        if s in successes:
+            ordered.append((s, successes[s]))
+        if len(ordered) >= LIMIT_TICKERS:
+            break
+    successes = {s: r for s, r in ordered}
+    failed_count = LIMIT_TICKERS - len(successes)
+
+    duration = time.time() - start
+    completed = len(successes)
+    total = LIMIT_TICKERS
```

Ruff (0.14.3)

Local variable `total` is assigned to but never used

Remove assignment to unused variable `total`

(F841)